examples/bosonic_tutorial_sampling.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A comparison of the asymptotic homodyne distributions and sampled values,
+for GKP and cat states."""
+
+import strawberryfields as sf
+import numpy as np
+import pylab as plt
```

Comment:
I've never heard of PyLab before. It seems like it's not recommended to use since it imports both `matplotlib.pyplot` and `numpy` into the same namespace.
```suggestion
import matplotlib.pyplot as plt
```
Is probably better, since `np` is imported above anyway.